1-exercises/J-string-substring/exercise.js

```diff
@@ -6,7 +6,7 @@
 
 let statement = "I like programming and dogs";
 
-statement = statement.substring();
+statement = statement.substring(0, 18);
```

Comment:
It will be better if the location could be calculated by indexOf dynamically

Reply:
Ah yes, that makes more logical sense rather than hardcode indexes. Thank you